frontend/src/scenes/surveys/SurveyEdit.tsx

```diff
@@ -2,8 +2,7 @@ import './EditSurvey.scss'
 
 import { DndContext } from '@dnd-kit/core'
 import { SortableContext, verticalListSortingStrategy } from '@dnd-kit/sortable'
-import { IconInfo } from '@posthog/icons'
-import { IconLock, IconPlus, IconTrash } from '@posthog/icons'
+import { IconInfo, IconLock, IconPlus, IconTrash } from '@posthog/icons'
```

Comment:
did you intend to add `IconInfo` here?  I think this might cause CI to yap because we're not using it afaict.

Reply:
i didn't intend to touch this! but I think what happened here is that my editor combined those two lines when formatting. if you see, ItemInfo was always there, and it's used in the file, but before we were importing it in a separate line